database_manager/src/lib.rs

```diff
@@ -200,11 +219,33 @@ pub fn inspect_db<E: EthSpec>(
             InspectTarget::ValueTotal => {
                 total += value.len();
             }
+            InspectTarget::Values => {
+                let base_path = &inspect_config.output_dir;
+                let file_path =
+                    base_path.join(format!("{}_{}.ssz", inspect_config.column.as_str(), key));
+
+                let write_result = fs::OpenOptions::new()
+                    .create_new(true)
+                    .write(true)
+                    .open(&file_path)
```

Comment:
I also think overwriting any existing file is fine. `File::create` should be sufficient for this

Reply:
Done, thanks!